src/mapdata.h

```diff
@@ -301,6 +301,7 @@ struct furn_t : map_data_common_t {
     furn_str_id open;  // Open action: transform into furniture with matching id
     furn_str_id close; // Close action: transform into furniture with matching id
     std::string crafting_pseudo_item;
+    std::string deployed_item; // item id string used to create furniture
```

Comment:
This should be a `itype_id`, not a `std::string` (it's the id of an item type, not an arbitrary string).